### ISSUE TITLE
Unable to render plugin on page if Geolocation Service set to none

### ISSUE THREAD

**Issue by T-Ivings** [doublesecretagency/craft-googlemaps#72]
In our CMS, we have our Geolocation Service set to none. We get the following error if we try to view the page using the plugin.

```
TypeError
doublesecretagency\googlemaps\helpers\GeolocationHelper::_getApiModel(): 
Argument #1 ($selected) must be of type string, null given, called in 
/srv/redacted/vendor/doublesecretagency/craft-googlemaps/src/helpers/GeolocationHelper.php on line 47
```

It functions properly for us if we change the service to ipstack and leave the API blank.

**Comment by lindseydiloreto** [comment]
Interesting, that definitely sounds like a bug. Thanks for reporting it! 👍 

Let me dig in and I'll let you know what I find.

**Comment by lindseydiloreto** [comment]
This is fixed, and will be available in the next release. Thanks for filing this issue! 🍺 

If you'd like to get the latest `dev` version now, change your `composer.json` file to say...

```json
"doublesecretagency/craft-googlemaps": "dev-v4-dev"
```

... and then run `composer update`.

**Comment by lindseydiloreto** [comment]
This patch has been officially released in **v4.3**. 👍
